contentful/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# CHANGELOG - Contentful
+
+<!-- towncrier release notes start -->
```

Comment:
@mauneelsorathia-crest @bparmar-crest This integration is only for the tile, right? I.e. there won't be any logs-related configuration or python code, right?

If that's the case, please remove this "towncrier" line and the CI will pass.

Reply:
@iliakur,
Thanks for your review.
Yes, there is no configuration related to log and no python code for this integration.
We are removing that line and pushing the changes.